app/services/construct_cap_attributes.rb

```diff
@@ -41,33 +39,18 @@ def product_name_and_description
       #{auction.summary})
   end
 
-  def link_to_product
-    auction.delivery_url
-  end
-
   def justification
     "Tock line item: #{auction.billable_to}"
   end
 
   def cost_per_unit
-    winning_bid.amount
+    auction.start_price
```

Comment:
So we bill the auction start price, regardless of how much it actually sold for?